src/PrestaShopBundle/Command/UpdateSchemaCommand.php

```diff
@@ -154,14 +154,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
             if (preg_match('/ALTER TABLE (.+?) /', $sql, $matches)) {
                 $tableName = $matches[1];
                 $matches = [];
-                if (preg_match_all('/([^\s,]*?) CHANGE (.+?) (.+?)(,|$)/', $sql, $matches)) {
-                    foreach ($matches[2] as $matchKey => $fieldName) {
+
+                $findChange = strpos($sql, 'CHANGE');
```

Comment:
It's not a good idea.
It will be true if the SQL request contains the word CHANGE, so
`ALTER TABLE nom_table RENAME COLUMN previous_name TO CHANGED_NAME`

You have to fix the regex instead

Reply:
I have change the way to make it work